### ISSUE TITLE
Cannot update the VSCodium on Windows.

### ISSUE THREAD

**Issue by charleech** [VSCodium/vscodium#1516]
**Describe the bug**

Cannot update the VSCodium on Windows from version 1.77.3 (MSI installation) to the latest version (In this case, version 1.78.2)

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Click the `Help` > `Check for updated`
2.  There is nothing happened.
3. Look at the `Help` again, found that there is a `Download Update`. 
4. Together, the `Gear` icon at the bottom left display notification as `1`
4.1  Click that  `Gear` icon  and found that there is a `Download Update (1)`
5. Either click on 3) or 4), the VSCodium open the web browser and start downloading the zip, in my case it is a `VSCodium-win32-x64-1.78.2.23132.zip`

**Expected behavior**
The VSCodium should update to the latest version.

**Screenshots**
![001](https://github.com/VSCodium/vscodium/assets/1825591/a1f53a69-3dbc-4599-8dd3-304c5252cc1a)
![002](https://github.com/VSCodium/vscodium/assets/1825591/051b1140-d833-4f3b-9da8-c6da644ff9c1)
![003](https://github.com/VSCodium/vscodium/assets/1825591/d8346329-47da-4486-8d56-587c963eddc8)

**Desktop (please complete the following information):**
 - OS: Windows 11 Home Single Language 22H2
 - Architecture  x64
 - Version 1.77.3 (MSI Installation)
 - App Manager N/A
 - Sandboxed N/A

**Additional context**
N/A

**Comment by daiyam** [comment]
Windows updates are tricky. The only way I found to detect a MSI install is to check if the app installed in a `Program Files` directory.

In which directory is it installed on your machine?

**Comment by charleech** [comment]
> In which directory is it installed on your machine?

I install in the another directory called `C:\My.Application\VSCodium`

Should I uninstall and reinstall to the `C:\Program Files` ?

**Comment by daiyam** [comment]
To update, download the `.msi` from https://github.com/VSCodium/vscodium/releases/tag/1.78.2.23132

**Comment by SamHobbsOrg** [comment]
I do not know if this is the same problem or not.

I was able to update VS Code, no problem.

VSCodium said there is an update. I closed VSCodium and executed it again. I assumed that would update VSCodium but it still said there is an update. So then I clicked on the link to update and restart. The first window hung, the one shown below.

I have a screenshot but my system is also hung. I will post this then restart my system, hopefully that will fix everything.
